recipes/pyjnius/activate.sh

```diff
@@ -0,0 +1,2 @@
+export PYJNIUS_JAR_BACKUP=$PYJNIUS_JAR
+export PYJNIUS_JAR=$CONDA_PREFIX/share/pyjnius/pyjnius.jar
```

Comment:
Minor comment: Would be good to quote these paths in the event the user has spaces in them.